contrib/storage-http/README.md

```diff
@@ -15,6 +15,7 @@ To configure the plugin, create a new storage plugin, and add the following conf
 {
   "type": "http",
   "cacheResults": true,
+  "useLegacyRequestParamSyntax": false,
```

Comment:
See comment.  I'd like to make the default behavior what Drill currently does and have the user explicitly enable this functionality.  Could we rename this param `advanced` or `enhanced` and thus change the boolean to `enableEnhancedParamSyntax`?

Reply:
Sure, I'll finish it soon.